src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -120,6 +133,12 @@ export class Lilies extends CoreGauge {
 
 	}
 
+	private checkAfflatusAvailability() {
+		if (!this.lilyGauge.empty && !this.bloodLilyGauge.capped) {
```

Comment:
This should also exclude hard casts when the player has the Thin Air status active - if Thin Air is active then there's no MP difference between the hard cast and the lily skill, and we ding elsewhere if someone is overcapping lilies.

See [conversation on Discord](https://discord.com/channels/441414116914233364/470049935077933076/1109949854223966289) for more info.